src/posix.mak

```diff
@@ -266,8 +266,8 @@ endif
 endif
 
 # Unique extra flags if necessary
-DMD_FLAGS  := -I$(ROOT) -Wuninitialized
-GLUE_FLAGS := -I$(ROOT) -I$(TK) -I$(C)
+DMD_FLAGS  := -I$(D) -I$(ROOT) -Wuninitialized
+GLUE_FLAGS := -I$(D) -I$(ROOT) -I$(TK) -I$(C)
```

Comment:
The common style of this Makefile is to skip the parentheses for one letter variables, i.e. `$D`

Reply:
All I have to say is `$(C)`.